MusicBandSimulator/MusicBandSimulator/Violin.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MusicBandSimulator
+{
+    /// <summary>
+    /// Об'єкт - скрипка. Імплементує інтерфейс IMusicInstrument.
+    /// </summary>
+    internal class Violin : IMusicInstrument
+    {
+        public string Name { get; set; }
```

Comment:
remove seter, so nobody can change the name of instrument

Reply:
done